README.md

```
@@ -1,96 +1,76 @@
 
 ### Implementation
 For Software:
 # Installation
-[commands]
+Save the code as index.html and open it in any modern web browser.
 
 # Run
-[commands]
+ Enter RPM and time (in minutes), then click Run Simulation to see rotations and totals.
 
 ### Project Documentation
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)